### DISCUSSION TITLE
Page Manager & Storage Manager Relationship Clarifications

### DISCUSSION THREAD

**Discussion by Stage4000** [GrapesJS/grapesjs#3876] [Q&A] [answered]
Can someone please help me understand how page manager and storage managers relationship works? Will storage manager automatically save all of the pages or do I need to make my back end save the pages individually? 

Sorry if this seems like a really stupid question.

**Comment by Ju99ernaut** [comment, marked as answer]
The page manager will automatically store all pages, you just need to setup your backend to recieve the payload from the editor.

**Reply by Stage4000** [reply]
Thank you

**Reply by Stage4000** [reply]
Do you happen to have an example of page manager in action? Im clearly not understanding something here.

Im using the webpage preset with a couple of other addons, I have remote saves set up and tested/verified working and I can clearly see that its sending a pages variable but when I run a ZIP export its only exporting the page im on. Im in testing phase right now so I only have the pages set in the grapes init as its shown in the docs so I can get a feel for how it works. What am I missing here?

**Reply by Stage4000** [reply]
You know what, I just stumbled on a Grapes project manager on some Genius' github page, I think im going to make use of it. <3

**Reply by Stage4000** [reply]
Uncaught TypeError: t.loadAll is not a function
    at n.value (project-manager.min.js:2)
    at n.value (project-manager.min.js:2)
    at run (project-manager.min.js:2)
    at r.callRun (grapesjs:2)
    at Object.runCommand (grapesjs:2)
    at r.updateActive (grapesjs:2)
    at m (grapesjs:2)
    at v (grapesjs:2)
    at d (grapesjs:2)
    at r.l.trigger (grapesjs:2)
    
    --EDIT--
    Ignore this, was due to improper setup on my end. Thank you for this amazing plugin, it should be part of the core development imho.

**Reply by throne1986** [reply]
> Uncaught TypeError: t.loadAll is not a function at n.value (project-manager.min.js:2) at n.value (project-manager.min.js:2) at run (project-manager.min.js:2) at r.callRun (grapesjs:2) at Object.runCommand (grapesjs:2) at r.updateActive (grapesjs:2) at m (grapesjs:2) at v (grapesjs:2) at d (grapesjs:2) at r.l.trigger (grapesjs:2)
> 
> ```
> --EDIT--
> Ignore this, was due to improper setup on my end. Thank you for this amazing plugin, it should be part of the core development imho.
> ```

Hii I am getting the same error ,  can you tell me how did you solve this issue?
